install_config/aggregate_logging.adoc

```diff
@@ -205,6 +218,10 @@ when `openshift_logging_use_ops` is set to `true`.
 |`openshift_logging_kibana_memory_limit`
 |The amount of memory to allocate to Kibana.
 
+|`openshift_logging_kibana_proxy_image`
+|The image version for the Kibana proxy.
+For example: `openshift_logging_kibana_proxy_image=<registry_url>:5000/openshift3/logging-auth-proxy:v3.X.X`
```

Comment:
Shall we use offical images as an example? registry.redhat.io/openshift3/oauth-proxy:v3.11. logging-auth-proxy had been deprecated in v3.11